rails.lua

```diff
@@ -101,4 +101,37 @@ minetest.register_craft({
 		{"default:steel_ingot", "group:stick", "default:steel_ingot"},
 		{"default:steel_ingot", "default:coal_lump", "default:steel_ingot"},
 	}
-})
\ No newline at end of file
+})
+
+boost_cart:register_rail(":carts:startstoprail", {
```

Comment:
Rename to `boost_cart:startstoprail` since this is no part of the official carts mod.